packages/main/src/RadioButton.hbs

```diff
@@ -15,7 +15,7 @@
 	<div class='ui5-radio-inner {{classes.inner}}'>
 		<svg class="ui5-radio-svg" focusable="false" aria-hidden="true">
 			<circle class="ui5-radio-svg-outer" cx="50%" cy="50%" r="50%" />
-			<circle class="ui5-radio-svg-inner" cx="50%" cy="50%" r="22%" />
+			<circle class="ui5-radio-svg-inner" cx="50%" cy="50%" r="25%" stroke-width="10"/>
```

Comment:
25% makes the "inner" blue dot grow to 11px.
The dot is specified to be 10px.

I also don't see any difference with `stroke-width="10"` and without it. Why is it needed? 
If it is necessary, it could be part of the CSS.